MAVEN.md

```diff
@@ -0,0 +1,34 @@
+#Publishing to maven.
+
+Publishing the kotlin native artifacts borrows mechanisms introduced in Gradle Native support, e.g. gradle metadata feature,
+thus there are some additional steps are required. First of all gradle version it shouldn't be less then gradle version of 
+kotlin native plugin dependens on (currently gradle-4.7). before gradle-5.0 feature GRADLE_METADATA should be enabled for build.
```

Comment:
I think it make sense to link a URL to some kind of Gradle docu where the feature "GRADLE_METADATA" is - well - documented...?!